src/main.cpp

```
@@ -260,7 +260,12 @@ int main(int argc, char** argv) {
         auto result = analyze(context);
         if (!quiet) {
             if (verbosity.print_invariants) {
-                print_invariants(std::cout, context.program, result, verbosity);
+                if(result.failed && !failure_slice) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)